src/shared/components/ui/Button/Button.stories.tsx

```diff
@@ -1,52 +1,65 @@
 import type { Meta, StoryObj } from "@storybook/react";
 
-import { Button } from "./Button";
+import { Button } from ".";
 
 const meta: Meta<typeof Button> = {
   component: Button,
-  parameters: {
-    componentSubtitle: "A simple button component",
+  parameters: { componentSubtitle: "A simple button component" },
+  args: {
+    variant: "default",
+    size: "default",
+    children: "Click Me",
+    disabled: false,
```

Comment:
Why passing values to variant, size and disabled which I assume they have default values in the component?

Reply:
ive seen it as a common practice to start with those arguments and override as needed to avoid having to go to the actual component